docs/logicapps-ai-course/02_build_conversational_agents/05-add-user-context-to-tools.md

```diff
@@ -48,56 +48,42 @@ Many solutions use mixed authorization methods. For example, a solution might re
 
 In conversational agent workflows, support for OBO authorization applies only to *shared* managed connectors that use delegated, per-user connections to work with Microsoft services or systems.
 
+> [!WARNING]
```

Comment:
This is "note" level information. A "warning" is reserved for scenarios where the customer can suffer injury or harm. A "caution" applies to situations where you can lose or damage data.

```suggestion
> [!NOTE]
```